lib/ReactViews/Feedback/FeedbackForm.tsx

```diff
@@ -436,6 +436,7 @@ const FormWrapper = styled(Box).attrs(props => ({
   styledWidth: "350px",
   backgroundColor: props.theme.textLight
 }))`
+  z-index: ${props => props.theme.notificationWindowZIndex};
```

Comment:
Move `FeedbackForm` above Data catalog (this is needed if you press "send feedback" in the new `TerriaError` `WarningBox`)